private/buf/cmd/buf/command/registry/registrylogin/registrylogin.go

```diff
@@ -53,9 +53,9 @@ func NewCommand(
 ) *appcmd.Command {
 	flags := newFlags()
 	return &appcmd.Command{
-		Use:   name + " <domain>",
+		Use:   name + " [domain]",
```

Comment:
Since this is an optional argument I think it should be in square brackets, not angle brackets.